woocommerce/wc-admin
# Removed not used `OnboardingEmailMarketing` reference

This is a follow-up for [PR 7595](https://github.com/woocommerce/woocommerce-admin/pull/7595) and PR 8398 (see [this comment](https://github.com/woocommerce/woocommerce-admin/pull/8398#discussion_r819059731)).

This PR deletes the importation of `OnboardingEmailMarketing` in `Events.php` since it's not being used.

(no changelog)

### Detailed test instructions:

- Check out this branch.
- Run the`wc_admin_daily` job.
- Verify that nothing is broken.

### ~UPDATE~
~Since currently [there is an error](https://github.com/woocommerce/woocommerce-admin/issues/8444) with the `wc_admin_daily` job, this PR should be rebased after having a fix in place.~
